.github/actions/setup-rocm/action.yml

```
@@ -57,6 +57,26 @@ runs:
             exit 1
         fi
 
+    - name: Runner health check diskspace
+      if: always()
+      shell: bash
+      run: |
+        diskspace_cutoff=70
+        diskspace=$(df -H / --output=pcent | sed -n 2p | sed 's/%//' | sed 's/ //')
+        msg="Please file an issue on pytorch/pytorch reporting the faulty runner. Include a link to the runner logs so the runner can be identified"
+        if [[ $diskspace -ge diskspace_cutoff ]] ; then
```

Nit (usually there are no need for space between condition and semicolon)
```suggestion
        if [[ $diskspace -ge diskspace_cutoff ]]; then
```